src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/task/AutoRejectVisitRequestsTask.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.task
+
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.visitscheduler.config.ExpiredApplicationTaskConfiguration.Companion.LOCK_AT_LEAST_FOR
+import uk.gov.justice.digital.hmpps.visitscheduler.config.ExpiredApplicationTaskConfiguration.Companion.LOCK_AT_MOST_FOR
+import uk.gov.justice.digital.hmpps.visitscheduler.service.VisitRequestsService
+
+@Component
+class AutoRejectVisitRequestsTask(private val visitRequestsService: VisitRequestsService) {
+
+  companion object {
+    private val LOG: Logger = LoggerFactory.getLogger(this::class.java)
+  }
+
+  // Every day at 8:00 PM
+  @Scheduled(cron = "0 0 20 * * *")
```

Comment:
could we have this as an env variable just in case we need to tweak it for different envs?